CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#6256](https://github.com/rubocop-hq/rubocop/pull/6256): Fix false positive in `Naming/FileName`. ([@sinsoku][])
```

Comment:
Would be great if there was one or two words about what the false positive was. 🙂

Reply:
I added a word about dotfiles to CHANGELOG. 📝